observability/micrometer/src/main/resources/schema/wildfly-micrometer_1_0.xsd

```diff
@@ -26,17 +26,25 @@
     <!-- The subsystem root element -->
     <xs:element name="subsystem">
         <xs:complexType>
-            <xs:attribute name="security-enabled" type="xs:boolean" default="true">
+            <xs:attribute name="exposed-subsystems" type="xs:string">
                 <xs:annotation>
                     <xs:documentation>
-                        True if authentication is required to access the HTTP endpoint on the HTTP management interface.
+                        The names of the subsystems (separated by spaces) that exposes their metrics in the vendor scope
+                        (or '*' to expose any subsystem metrics).
```

Comment:
Trivial, but in this situation, we should just keep these descriptions in sync with the `localdescriptions.properties`.